sync/src/main/java/tech/pegasys/teku/sync/OutOfOrderException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.sync;
+
+import com.google.common.primitives.UnsignedLong;
+import tech.pegasys.teku.networking.eth2.rpc.core.InvalidResponseException;
+
+public class OutOfOrderException extends InvalidResponseException {
```

Comment:
Would be helpful to make it clear this error is specific to the BlocksByRange response:
```suggestion
public class BlocksByRangeResponseOutOfOrderException extends InvalidResponseException {
```

Reply:
Done.